### PR TITLE
fix: remove border from high contrast mode to make focus more visible

### PR DESCRIPTION
**Pull request by smhigley**
## Previous Behavior

There was a transparent border on the AccordionHeader button, which showed up in high contrast mode (since the transparent color is overridden). This made the focus outline barely visible (second item is focused):

screenshot of the docs accordion example in high contrast dark mode, with a slight difference in the outline of the second item

## New Behavior

There is no 1px border, so the focus outline in HCM is clearly visible, and the accordion styles match the appearance outside of forced-colors mode. This doesn't change the accordion height, since it already has a `44px` height style.

screenshot of the same example, with no borders around accordions and a clearly visible focus outline

## Related Issue(s)

- Fixes [ADO issue](https://dev.azure.com/microsoftdesign/fluent-ui/_workitems/edit/18624)

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30865/perf-test-react-components/Avatar-mount_base.html">658</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30865/perf-test-react-components/Avatar-mount.html">641</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30865/perf-test-react-components/Button-mount_base.html">322</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30865/perf-test-react-components/Button-mount.html">305</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30865/perf-test-react-components/Field-mount_base.html">1129</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30865/perf-test-react-components/Field-mount.html">1150</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30865/perf-test-react-components/FluentProvider-mount_base.html">684</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30865/perf-test-react-components/FluentProvider-mount.html">720</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30865/perf-test-react-components/FluentProviderWithTheme-mount_base.html">80</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30865/perf-test-react-components/FluentProviderWithTheme-mount.html">89</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30865/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">60</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30865/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">63</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30865/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">70</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30865/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">83</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30865/perf-test-react-components/MakeStyles-mount_base.html">851</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30865/perf-test-react-components/MakeStyles-mount.html">874</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30865/perf-test-react-components/Persona-mount_base.html">1728</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30865/perf-test-react-components/Persona-mount.html">1718</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30865/perf-test-react-components/SpinButton-mount_base.html">1420</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30865/perf-test-react-components/SpinButton-mount.html">1390</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports | Baseline (minified/GZIP) | PR    | Change     |
| :---------------- | -----------------------: | ----: | ---------: |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr> | `100.125 kB`<br />`30.423 kB` | `99.599 kB`<br />`30.264 kB` | `-526 B` <br />`-159 B` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `220.047 kB`<br />`62.171 kB` | `219.521 kB`<br />`62.081 kB` | `-526 B` <br />`-90 B` |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports | Size (minified/GZIP) |
| ----------------- | -------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr> | `71.098 kB`<br />`20.515 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr> | `43.585 kB`<br />`14.352 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr> | `7.944 kB`<br />`2.588 kB` |
</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/0a593445b3809289313bc6d49dedd18f33a850e4'>0a593445b3809289313bc6d49dedd18f33a850e4</a></sub>
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-accordion-7cc895ca-0a54-47c7-bbf4-7ca1d1553d65.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-accordion-7cc895ca-0a54-47c7-bbf4-7ca1d1553d65.json:1]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30865&commitId=bd068d60ed7d27b9f13418cc97c665bf260dcbc2&env=prod&clientType=FLUENTUI)


  <details><summary><b>Accordion Converged</b> 25 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion Converged.disabled - High Contrast.normal.chromium.png|946| Changed|
|Accordion Converged.expandIcon   Icon-   - RTL.normal.chromium.png|1045| Changed|
|Accordion Converged.expandIcon   Icon-   - Dark Mode.normal.chromium.png|1017| Changed|
|Accordion Converged.icon   Icon-  .normal.chromium.png|690| Changed|
|Accordion Converged.disabled.normal.chromium.png|364| Changed|
|Accordion Converged.visibility+focus - Dark Mode.focus opened.chromium.png|271| Changed|
|Accordion Converged.icon   Icon-   - RTL.normal.chromium.png|694| Changed|
|Accordion Converged.disabled - Dark Mode.normal.chromium.png|271| Changed|
|Accordion Converged.visibility+focus - Dark Mode.normal.chromium.png|271| Changed|
|Accordion Converged.expandIcon   Icon-  .normal.chromium.png|1044| Changed|
|Accordion Converged.expandIconPosition  end .normal.chromium.png|278| Changed|
|Accordion Converged.size.normal.chromium.png|665| Changed|
|Accordion Converged.visibility+focus - High Contrast.focus opened.chromium.png|324| Changed|
|Accordion Converged.visibility+focus - High Contrast.focus closed.chromium.png|997| Changed|
|Accordion Converged.expandIconPosition  end  - RTL.normal.chromium.png|280| Changed|
|Accordion Converged.visibility+focus - Dark Mode.focus closed.chromium.png|1095| Changed|
|Accordion Converged.expandIcon   Icon-   - High Contrast.normal.chromium.png|1214| Changed|
|Accordion Converged.size - RTL.normal.chromium.png|653| Changed|
|Accordion Converged.visibility+focus - RTL.normal.chromium.png|281| Changed|
|Accordion Converged.visibility+focus.focus closed.chromium.png|1260| Changed|
|Accordion Converged.visibility+focus - RTL.focus opened.chromium.png|281| Changed|
|Accordion Converged.visibility+focus - High Contrast.normal.chromium.png|324| Changed|
|Accordion Converged.visibility+focus - RTL.focus closed.chromium.png|1263| Changed|
|Accordion Converged.visibility+focus.focus opened.chromium.png|279| Changed|
|Accordion Converged.visibility+focus.normal.chromium.png|279| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Comment (behowell) [reply change/@fluentui-react-accordion-7cc895ca-0a54-47c7-bbf4-7ca1d1553d65.json:1]: LGTM

**Review by behowell — APPROVED** [review]

**Review by sopranopillow — APPROVED** [review]

**Pull request merged**